README.md

```diff
@@ -10,9 +10,6 @@ This includes a fully [RFC 7946](https://tools.ietf.org/html/rfc7946)-compliant
 
 Most of the implementation is a direct translation from [turf.js](https://github.com/Turfjs/turf).
 
-## GeoJSON Object Model
```

Comment:
where has this gone to? I think it's really important to explain that to users of the lib